### PR TITLE
fix AttachmentTestCase leaving a deployment around causing BasicOperationsUnitTestCase#testPathInfo to measure size of the content dir > 0b

### PR DESCRIPTION
**Pull request by luck3y**
@jfdenise, fyi, I think this only shows up if the test order is randomly different.

### CODE REVIEW

**Comment by bstansberry** [comment]
This kind of cleanup is good, so the PR is fine with me if ok with Jeff. But BasicOperationsUnitTestCase failing any time there is data in the content dir seems overly fragile.

**Comment by luck3y** [comment]
We could clean that up in @Before in BOUTC maybe, as long as those aren't sharing configs (Which I bet they aren't.)

Yah, I'll add that too quick, might save someone some time later.

**Comment by jfdenise** [comment]
Thank you for the cleanup, test change is fine.
JF

File: testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/BasicOperationsUnitTestCase.java [hunk 1]
```diff
@@ -112,6 +115,38 @@
...
+
+            for (File f : file.listFiles()) {
+                if (f.isFile()) {
+                    f.delete();
+                } else if (f.isDirectory()) {
+                    FileUtils.deleteDirectory(f);
+                }
+            }
+        }
+    }
```
Comment (bstansberry) [inline testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/BasicOperationsUnitTestCase.java]: This is likely just as fragile. If a test has left content in this dir it probably has left config referencing it in the model and once this runs any subsequent reload will blow up.

I don't see a significant benefit to this test asserting that this dir has a size of zero. At least not any benefit that justifies complexity.

Comment (luck3y) [inline testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/BasicOperationsUnitTestCase.java]: Ah, OK, I'd interpreted this as something the test really wanted to verify. I'll remove that and my last change and squash.

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

**Comment by bstansberry** [comment]
@luck3y needs a rebase

**Comment by luck3y** [comment]
@bstansberry Should be good to go now.

**Comment by bstansberry** [comment]
retest this please

**Pull request merged**